apps/src/storage/dataBrowser/DataVisualizer.jsx

```diff
@@ -272,7 +231,16 @@ class DataVisualizer extends React.Component {
             </div>
           )}
         </div>
-        <div id="chart-area" />
+        {this.state.chartType === 'Cross Tab' ? (
+          <CrossTabChart
+            parsedRecords={this.state.parsedRecords}
+            numericColumns={this.state.numericColumns}
+            rowName={this.state.xValues}
+            columnName={this.state.yValues}
+          />
+        ) : (
+          <div id="chart-area" />
+        )}
```

Comment:
An early refactoring thought: `DataVisualizer` seems like it may continue to get more complex over time.  It might be a good time to extract a `Chart` component you could render here that encapsulates the "how do I render this chart type" detail, including your `updateChart()` method, so it's separated from the controls and their related business logic.